stories/assets/js/parallax_lines.js

```diff
@@ -0,0 +1,25 @@
+import './viewport';
+import { breakpoints } from './global-variables';
+
+export function parallaxlines() {
+  $(window).scroll(() => {
+    var $sticky = $('.ourexpertise .glide__slides');
+    var stickyTop = $sticky.offset().top;
+    var windowTop = $(window).scrollTop();
+    var diff = windowTop + 500;
+    $('.ourexpertise .glide__slides').each(function () {
+      if ($(window).width() > breakpoints.small) {
```

Comment:
Use variable declaration for `$(window).width()`. Refrain from using it inside of condition. 
Eg.:
```
let windowWidth = $(window).width();
if (windowWidth > breakpoints.small) {
  // Do something...
}
```